### PR TITLE
Django 2.0 compatibility

### PR DESCRIPTION
**Pull request by tdruez**
- Adds `on_delete` argument on `user` FK field #46

### CODE REVIEW

**Comment by avelis** [comment]
@tdruez Thanks for adding this change for Django 2.0 compatibility.

File: .travis.yml [hunk 1]
```diff
@@ -33,6 +34,8 @@ matrix:
       env: DJANGO_VERSION="1.10"
     - python: "3.3"
       env: DJANGO_VERSION="1.11"
+    - python: "3.3"
```
Comment (fengsi) [inline .travis.yml]: Why would we need Python 3.3 for Django 2.0?
https://docs.djangoproject.com/en/2.0/releases/2.0/#python-compatibility

Comment (avelis) [reply .travis.yml]: @fengsi We could drop it from the testing matrix to be excluded for Django 2.0

Comment (tdruez) [reply .travis.yml]: @fengsi we don't, this is an `exclude` list ;)

Comment (avelis) [reply .travis.yml]: @fengsi If you look on [line 26](https://github.com/zapier/django-rest-hooks/blob/master/.travis.yml#L26). Python 3.3 is added as part of the exclusion for Django 2.0

Comment (fengsi) [reply .travis.yml]: my bad 👍

**Pull request merged**